core/embed/bootloader_ci/memory.ld

```diff
@@ -23,6 +23,9 @@ ccmram_end = ORIGIN(CCMRAM) + LENGTH(CCMRAM);
 sram_start = ORIGIN(SRAM);
 sram_end = ORIGIN(SRAM) + LENGTH(SRAM);
 
+/* IMAGE_HEADER_SIZE is 0x400, this is for interaction-less firmware update start */
+firmware_header_start = ccmram_end - 0x400;
```

Comment:
`bootloader_ci` still cleans the header on startup. not sure we need the functionality there though

Reply:
Fixed as discussed - `bootloader_ci` may be used also to test this non-interactive update now.